pkg/builds/schema_doc.go

```diff
@@ -297,7 +297,8 @@ var generatedSchemaJSON = `{
     "coreos-assembler.meta-stamp",
     "coreos-assembler.overrides-active",
     "coreos-assembler.yumrepos-git",
-    "coreos-assembeler.oci-imported",
+    "coreos-assembler.oci-labels",
```

Comment:
part of me reads this and things these are the labels from the COSA container that was used to do the build.

Does anyone else mistakenly read it that way?

Reply:
Indeed. Happy to tweak the key name here. I guess the idea was to have it be consistent with `coreos-assembler.oci-imported` to underline that they're related.

Reply:
`coreos-assembeler.oci-imported-labels` ?